controller-server/src/main/java/com/yahoo/vespa/hosted/controller/athenz/filter/UserAuthWithAthenzPrincipalFilter.java

```diff
@@ -29,7 +28,6 @@
  *
  * @author bjorncs
  */
-@Provides("UserAuthWithAthenzPrincipalFilter")
```

Comment:
Does it override the default Provides (full class name)?

Reply:
Okay. I will fix that later on. First I want to try this out asap.